src/theory/quantifiers/sygus/synth_conjecture.h

```diff
@@ -174,6 +182,46 @@ class SynthConjecture
   SygusModule* d_master;
   //------------------------end modules
 
+  //------------------------enumerators
+  /**
+   * Get model values for terms n, store in vector v. This method returns true
+   * if and only if all values added to v are non-null.
+   */
+  bool getEnumeratedValues(std::vector<Node>& n, std::vector<Node>& v);
+  /**
+   * Get model value for term n. If n has a value that was excluded by
+   * datatypes sygus symmetry breaking, this method returns null.
+   */
+  Node getEnumeratedValue(Node n);
+  /** enumerator generators for each actively-generated enumerator */
+  std::map<Node, std::unique_ptr<EnumValGenerator> > d_evg;
+  /**
+   * Map from enumerators to whether they are currently being
```

Comment:
clarify that the map is from enum to the abstract value you are currently enumerating concrete values for, and that when it's null you're no longer in "actively-generating" state